### PR TITLE
Adding some additional verbiage to clarify the Gitpod instructions

### PR DESCRIPTION
**Pull request by craigatk**
## Which problem is this PR solving?

Adding a bit more detail for the Gitpod instructions so new users can easily run the app there.

## Short description of the changes

1. Added some more verbiage and a screenshot for accessing the app when it's running in Gitpod
2. Simplified the number of choices for running the app with Gitpod or running locally

## How to verify that this has the expected result

Use the instructions to deploy and run the app in Gitpod

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,21 +11,25 @@ Spoiler: this microservice implements the <a href="https://en.wikipedia.org/wiki
 Recommended:
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/honeycombio/intro-to-o11y-python)
 
-Alternative: [remix this app on Glitch](https://glitch.com/edit/#!/intro-to-o11y-python).
+Gitpod is a free cloud environment where you can run the example without needing to clone the code or install Python on your machine.
 
-Alternative: Clone and run locally.
+Or you can clone this repo and run the app locally.
```
Comment (cewkrupa) [inline README.md]:
```suggestion
You can also clone this repo and run the app locally.
```

File: README.md [hunk 2]
```diff
@@ -11,21 +11,25 @@ Spoiler: this microservice implements the <a href="https://en.wikipedia.org/wiki
...
  If you use [VSCode devcontainers](https://code.visualstudio.com/docs/remote/containers-tutorial),
 then reopen in a container. Otherwise, have python3 and pip installed; and run `pip3 install -r requirements.txt`.
```
Comment (cewkrupa) [inline README.md:18]: Do we care about the version of python beyond 3?

File: README.md [hunk 3]
```diff
@@ -11,21 +11,25 @@ Spoiler: this microservice implements the <a href="https://en.wikipedia.org/wiki
...
-`./run`
+Start the app by executing `./run` in the terminal.
 
 ### See the app
 
-Hit the app at http://localhost:5000
+If you are running the app in Gitpod, navigate to the "Ports" tab and click the address for port 5000 to open the app in a new tab:
 
-Activate the sequence of numbers by pushing **Go**. After you see numbers, push **Stop**. Try this a few times.
+![Gitpod open address](img/gitpod-ports.png "Gitpod open address")
```
Comment (cewkrupa) [inline README.md:28]: woooo screenshots :D

File: README.md [hunk 4]
```diff
@@ -11,21 +11,25 @@ Spoiler: this microservice implements the <a href="https://en.wikipedia.org/wiki
...
+
+Or if you are running locally, access the app at http://localhost:5000
```
Comment (cewkrupa) [inline README.md]:
```suggestion
If you are running locally, access the app at http://localhost:5000.
```

File: README.md [hunk 5]
```diff
@@ -11,21 +11,25 @@ Spoiler: this microservice implements the <a href="https://en.wikipedia.org/wiki
...
+
+Then activate the sequence of numbers by pushing the **Go** button. After you see numbers, push **Stop**. Try this a few times.
```
Comment (cewkrupa) [inline README.md]:
```suggestion
Activate the sequence of numbers by selecting the **Go** button.
After the app displays numbers, select **Stop**.
Try this a few times.
```

File: README.md [hunk 6]
```diff
@@ -35,22 +39,22 @@ Press `Ctrl-C` in the terminal where the app is running.
 
 ## Configure telemetry to connect to Honeycomb
 
-We need to set a few environment variables.
+We need to set a few environment variables to configure OpenTelemetry to send data to Honeycomb.
 
 Get a Honeycomb API Key from your Team Settings in [Honeycomb](https://ui.honeycomb.io).
 (find this by clicking on your profile in the lower-left corner.)
```
Comment (cewkrupa) [inline README.md]:
```suggestion
(find this by selecting your profile in the lower-left corner of the Honeycomb user interface.)
```

File: README.md [hunk 7]
```diff
@@ -35,22 +39,22 @@ Press `Ctrl-C` in the terminal where the app is running.
...
 
 Recommended: set up a `.env` file, and the app will read it.
 
-Copy the example: `cp .env.example .env`
+Copy the example env: `cp .env.example .env`
 
-Now, edit `.env` and put your own API key where it belongs. This file will be ignored by git,
+Now, edit `.env` and put your own Honeycomb API key where it belongs. This file will be ignored by git,
 so you won't accidentally commit your API key.
```
Comment (cewkrupa) [inline README.md]:
```suggestion
Edit `.env` and replace the placeholder value for HONEYCOMB_API_KEY with your own Honeycomb API key. 
This file will be ignored by git, so you will not accidentally commit your API key.
```

File: README.md [hunk 8]
```diff
@@ -61,13 +65,15 @@ You can name the Honeycomb Dataset anything you want.
...
+Start the app with `./run`
 
-`python3 server.py`
+If you are running the app in Gitpod, navigate to the "Ports" tab and click the address for port 5000 to open the app in a new tab:
 
-Hit it at http://localhost:5000
+![Gitpod open address](img/gitpod-ports.png "Gitpod open address")
 
-Activate the sequence of numbers.
+If you are running locally, access the app at http://localhost:5000
```
Comment (cewkrupa) [inline README.md:76]:
```suggestion
If you are running locally, access the app at http://localhost:5000.
```

File: README.md [hunk 9]
```diff
@@ -61,13 +65,15 @@ You can name the Honeycomb Dataset anything you want.
...
+
+Activate the sequence of numbers by pushing **Go**. After you see numbers, push **Stop**. Try this a few times.
```
Comment (cewkrupa) [inline README.md]:
```suggestion
Activate the sequence of numbers by selecting the **Go** button.
After the app displays numbers, select **Stop**.
Try this a few times.
```

**Review by cewkrupa — COMMENTED** [review]
This looks great! I added some suggestions based on the [docs styleguide](https://github.com/honeycombio/docs/blob/main/styleguide.md), let me know what you think

File: README.md (hunk 1 above)
Comment (craigatk) [reply README.md]: Nice, thanks for the suggestions!

File: README.md (hunk 2 above)
Comment (craigatk) [reply README.md:18]: Good question, I think we're good as long as it's Python 3

**Review by cewkrupa — APPROVED** [review]

**Pull request merged**